src/AWSMetadataUtilities.jl

```diff
@@ -95,16 +126,19 @@ Get the low-level definitions for all AWS Services.
 # Returns
 - `Vector{String}`: Array of low-level service code to be written into `AWSServices.jl`
 """
-function _generate_low_level_definitions(services::AbstractArray{OrderedDict})
+function _generate_low_level_definitions(services::AbstractArray)
```

Comment:
```suggestion
function _generate_low_level_definitions(services::AbstractArray{<:AbstractDict})
```
Since we use the dict interface below we can require this.